onnxruntime/core/providers/cuda/object_detection/non_max_suppression.h

```diff
@@ -5,7 +5,7 @@
 
 #include "core/common/common.h"
 #include "core/framework/op_kernel.h"
```

Comment:
>op_kernel.h" [](start = 25, length = 12)

can we also remove op_kernel.h from non_max_suppression.h and roialign.h?

Reply:
good idea. searched CUDA .h files for redundant op_kernel.h includes and removed them

---
In reply to: [542679109](https://github.com/microsoft/onnxruntime/pull/6118#discussion_r542679109) [](ancestors = 542679109)